.github/workflows/release.yml

```diff
@@ -52,6 +52,7 @@ jobs:
     environment: release
     permissions:
       contents: read
+      id-token: write
```

Comment:
An `id-token` permission is necessary: https://docs.npmjs.com/trusted-publishers#github-actions-configuration